pyrene/src/components/Button/Button.tsx

```diff
@@ -48,25 +48,24 @@ const Button: React.FC<ButtonProps> = ({
   onClick = () => null,
   label,
 }: ButtonProps) => (
-  <div styleName="buttonContainer">
+  <div className={styles.buttonContainer}>
     <button
       type="submit"
-      className="unSelectable"
-      styleName={
-        clsx('button',
-          { [`type-${type}`]: true },
-          { disabled: disabled },
-          { loading: loading })
+      className={
+        clsx('unSelectable', styles.button,
+          { [styles[`type-${type}`]]: true },
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

```
styles[`type-${type}`],
```